docs/excel/custom-functions-overview.md

```diff
@@ -72,32 +72,32 @@ Excel.run(function(ctx) {
 });
 ```
 
-**Registration** of the custom function takes place within the `Excel.Script.customFunctions["CONTOSO"]["ADD42"]` code block. You need the following parameters to register the function in Excel:
+**Registration** of the custom function uses the `Excel.Script.customFunctions["CONTOSO"]["ADD42"]` code block. You need the following parameters to register the function in Excel:
 
--   Prefix and function name: The first value in `Excel.Script.customFunctions` is the prefix (in this case, CONTOSO is the prefix). The second value in `Excel.Script.customFunctions` is the function name (in this case ADD42 is the function name). The prefix and the function name are separated using a period. To use your custom function, combine the function's prefix (CONTOSO) with the function's name (ADD42) and enter `=CONTOSO.ADD42` into a cell. By convention, prefixes and function names use upper case letters. The prefix is intended to be used as an identifier for your add-in.
--   call: Defines the JavaScript function to call (for example, add42). The name of the JavaScript function does not need to match the name that you register in Excel.
--   description: The description appears in the autocomplete menu in Excel.
--   helpUrl: When the user requests help for a function, Excel opens a task pane and displays the web page found at this URL.
--   result: The result types of your function can be either a number or string, and either a single value of group of values.
+-   Prefix and function name: The first value in `Excel.Script.customFunctions` is the prefix (in this case, CONTOSO is the prefix). The second value in `Excel.Script.customFunctions` is the function name (in this case ADD42 is the function name). In Excel, the prefix and the function name are separated using a period: to use your custom function, combine the function's prefix (CONTOSO) with the function's name (ADD42) and enter `=CONTOSO.ADD42` into a cell. By convention, prefixes and function names use upper case letters. The prefix is intended to be used as an identifier for your add-in.
+-   `call`: Defines the JavaScript function to call (for example, `add42`). The name of the JavaScript function does not need to match the name that you register in Excel.
+-   `description`: The description appears in the autocomplete menu in Excel.
+-   `helpUrl`: When the user requests help for a function, Excel opens a task pane and displays the web page found at this URL.
+-   `result`: Defines the type of information returned by the function to Excel.
 
-    -   resultType: Your function can return either a number or a string. For more information see &lt;&lt;LINK&gt;&gt;
+    -   `resultType`: Your function can return either a `"string"` or a `"number"` (also used for dates and currencies). For more information see &lt;&lt;LINK&gt;&gt;.
```

Comment:
Please remove the quotes from string and number. Thanks.

Reply:
The quotation marks are there to remind the reader that they're strings used in the resultType enums